osu.Game/Overlays/BeatmapSet/Scores/DrawableScore.cs

```diff
@@ -1,142 +1,217 @@
-                    Font = @"Exo2.0-RegularItalic",
-                    Margin = new MarginPadding { Left = side_margin }
+                    RelativeSizeAxes = Axes.Both,
+                    Alpha = 0,
                 },
-                new DrawableFlag(score.User.Country)
+                new DrawableScoreData(index, score, maxModsAmount),
+            };
+
+            if (index % 2 != 0)
+                background.Alpha = 0;
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(OsuColour colours)
+        {
+            hoveredBackground.Colour = colours.Gray4;
+            background.Colour = colours.Gray3;
+        }
+
+        protected override bool OnHover(HoverEvent e)
+        {
+            hoveredBackground.FadeIn(fade_duration, Easing.OutQuint);
+            return base.OnHover(e);
+        }
+
+        protected override void OnHoverLost(HoverLostEvent e)
+        {
+            hoveredBackground.FadeOut(fade_duration, Easing.OutQuint);
+            base.OnHoverLost(e);
+        }
+
+        protected override bool OnClick(ClickEvent e) => true;
+
+        private class ClickableScoreUsername : ClickableUserContainer
+        {
+            private readonly SpriteText text;
+            private readonly SpriteText textBold;
+
+            public ClickableScoreUsername()
+            {
+                Add(text = new SpriteText
                 {
                     Anchor = Anchor.CentreLeft,
                     Origin = Anchor.CentreLeft,
-                    Size = new Vector2(30, 20),
-                    Margin = new MarginPadding { Left = 60 }
-                },
-                new ClickableUsername
+                    TextSize = text_size,
+                });
+
+                Add(textBold = new SpriteText
                 {
                     Anchor = Anchor.CentreLeft,
                     Origin = Anchor.CentreLeft,
-                    User = score.User,
-                    Margin = new MarginPadding { Left = 100 }
-                },
-                modsContainer = new ScoreModsContainer
+                    TextSize = text_size,
+                    Font = @"Exo2.0-Bold",
+                    Alpha = 0,
+                });
+            }
+
+            protected override void OnUserChange(User user)
+            {
+                text.Text = textBold.Text = user.Username;
+            }
+
+            protected override bool OnHover(HoverEvent e)
+            {
+                textBold.FadeIn(fade_duration, Easing.OutQuint);
+                text.FadeOut(fade_duration, Easing.OutQuint);
+                return base.OnHover(e);
+            }
+
+            protected override void OnHoverLost(HoverLostEvent e)
+            {
+                textBold.FadeOut(fade_duration, Easing.OutQuint);
+                text.FadeIn(fade_duration, Easing.OutQuint);
+                base.OnHoverLost(e);
+            }
+        }
+
+        private class DrawableScoreData : ScoreTableLine
```

Comment:
Why is this suffixed as "data"? Probably just `DrawableScoreRow`? and base class should be a row too, because that's what it is.